src/Spice86.Core/Emulator/Devices/Video/AeonCard.cs

```diff
@@ -714,7 +714,7 @@ public class AeonCard : DefaultIOPortHandler, IVideoCard, IAeonVgaCard, IDisposa
                 throw new NotSupportedException($"Video mode {id} is not supported.");
         }
 
-        _loggerService.Information("Setting video mode to {@Mode}", id);
+        _loggerService.Verbose("Setting video mode to {Mode}", id);
```

Comment:
Video mode switches are not that frequent and have a big impact. I'd leave this at Information.

Reply:
Implemented.